program.html

```diff
@@ -0,0 +1,128 @@
+            build personal connections with the current brothers, and, most importantly, bond with the people they're in the program with. " />
+        <meta property="og:title" content="Lambda Theta Delta | Fraternity | Irvine">
+        <meta property="og:url" content="https://www.ltdfraternity.com/program.html">
+        <meta property="og:type" content="website" />
+        <meta property="og:site_name" content="Lambda Theta Delta" />
+        <meta property="og:locale" content="en_US" />
+        <meta property="og:image" content="imgs/background.jpg">
+        <meta name="twitter:card" content="summary_large_image" />
+        <meta name="twitter:title" content="Lambda Theta Delta | Fraternity | Irvine" />
+        <meta name="twitter:description"
+            content="After being offered and accepting a bid, members enter a 10 week membership development program that aims to offer future members professional insight, 
+            build personal connections with the current brothers, and, most importantly, bond with the people they're in the program with. " />
+
+        <meta name="viewport" content="width=device-width, initial-scale=1"/>
+
+        <script type="application/ld+json">
+            {
+                "@context": "https://schema.org/",
+                "@type": "Organization",
+                "logo": "https://upload.wikimedia.org/wikipedia/commons/3/33/Lambda_Theta_Delta_Fraternity_Crest_2012.png",
+                "areaServed": {
+                    "@type": "City",
+                    "name": "Irvine",
+                    "sameAs": "https://en.wikipedia.org/wiki/Irvine,_California"
+                },
+                "foundingDate": "1983-03-02",
+                "email": "ltdfraternity@gmail.com",
+                "url": "https://www.ltdfraternity.com",
+                "name": "Lambda Theta Delta",
+                "sameAs": "https://www.instagram.com/lambdathetadelta/",
+                "alternateName": "LTD",
+                "slogan": "Loyalty. Trust. Diversity.",
+                "description": "Lambda Theta Delta is the first established multicultural fraternity at the University of California, Irvine, and remains one of the most prominent fraternities within the Asian Greek network",
+                "keywords": "Asian Fraternity, Fraternity, LTD, Asian, Asian Greek, MSFC, Lambda, Theta, Delta"
+            }
+        </script>
+        <script src="schema.js"></script>
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    </head>
+
+    <body>
+        <script src="nav.js"></script> <!-- Navbar code kept in nav.js for reusability  -->
+        <div class="overlap">
+
+            <img class="elements img" src="https://static.wixstatic.com/media/625c2f_0661ba40f06241dcb1af2fdc1f83b762~mv2.jpg/v1/fill/w_1746,h_1244,al_c,q_90,usm_0.66_1.00_0.01,enc_auto/625c2f_0661ba40f06241dcb1af2fdc1f83b762~mv2.jpg">
```

Comment:
I would also consider making these 2 elements display inline-flex or some sort of inline, so they can be positioned on the same line. That way we can avoid the `position: relative;` in the css